apps/mobile/src/state/chat/atoms/showUserAvatarDetailAtom.ts

```diff
@@ -0,0 +1,4 @@
+import {atom} from 'jotai'
+import {type UriString} from '@vexl-next/domain/dist/utility/UriString.brand'
+
+export const userAvatarDetailAtom = atom<UriString | undefined>(undefined)
```

Comment:
Is there any reason why is this living here, instead of in molecule in `ChatDetailScreen/atoms/index.ts`? It makes more sense, since this is not a global state rather it is a state related to one chat.

Reply:
Also, it would be nice, if we could, in the future use this for displaying any image. Not just user avatar. I would suggest renaming this to `openedImageAtom` (and moving this to molecule as mentioned above)

Reply:
Resolved. Thanks for moving this to ChatDetailScreen atoms